store/mockstore/unistore/cophandler/analyze.go

```diff
@@ -116,7 +140,7 @@ func handleAnalyzeCommonHandleReq(dbReader *dbreader.DBReader, rans []kv.KeyRang
 	hg := statistics.HistogramToProto(processor.statsBuilder.Hist())
 	var cm *tipb.CMSketch
 	if processor.cms != nil {
-		cm = statistics.CMSketchToProto(processor.cms)
+		cm = statistics.CMSketchToProto(processor.cms, nil)
 	}
 	data, err := proto.Marshal(&tipb.AnalyzeIndexResp{Hist: hg, Cms: cm})
```

Comment:
Should we also convert `topNValuePars` to `PB` and return it here?

Reply:
common handle doesn't collect topn now. so we don't need to handle it.